### PR TITLE
Adds Travis and CodeClimate configuration for code analysis

### LINKED ISSUE #1: Add Travis testing framework

### PR DESCRIPTION
**Pull request by matt-bernhardt**
This closes #1 

Code review:
- [x] @PBruk 
- [x] @frrrances

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  - "**.js"
+  - "**.jsx"
+  - "**.php"
+  - "**.module"
+  - "**.inc"
+exclude_paths:
+- fonts/**/*
+- js/jquery.flexslider-min.js
+- js/jquery.flexslider.js
+- navwalker.php
```
Comment (frrrances) [inline .codeclimate.yml]: needs new line

File: .eslintrc [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+  - vars: all
+    args: after-used
+  no-use-before-define: 0
+  no-useless-call: 0
+  no-void: 0
+  no-var: 0
+  no-warning-comments:
+  - 0
+  - terms:
+    - todo
```
Comment (frrrances) [inline .eslintrc:118]: ooo nice! keywords... we should standardize around these

**Comment by frrrances** [comment]
looks good! 👍

File: .eslintrc (hunk 2 above)
Comment (matt-bernhardt) [inline .eslintrc:118]: Yeah, this is a boilerplate file that CodeClimate asks for - we can tune it, but I haven't gone in and poked around in great depth.

File: .codeclimate.yml (hunk 1 above)
Comment (matt-bernhardt) [inline .codeclimate.yml]: Fixed - I thought I got them all, good catch.

**Comment by PBruk** [comment]
LGTM. +1

**Pull request merged**